BoldGrid/post-and-page-builder
# adjust menu button classes to display correctly

Resolves #460 

### Testing Instructions ###
1. install test zip file.
[post-and-page-builder.zip](https://github.com/BoldGrid/post-and-page-builder/files/9802684/post-and-page-builder.zip)

2. Install Crio and Crio Premium
3. Create a Page footer with a menu and at least one menu item being a button.
4. Ensure the height of the buttons is correct, and that the buttons do not display the hover underline effects.
